lib/backburner/index.js

```diff
@@ -28,7 +28,7 @@ export default function Backburner(queueNames, options) {
 
   var _this = this;
   this._boundClearItems = function() {
-    clearItems();
+    clearItems.apply(_this, arguments);
```

Comment:
I guess I'd prefer to see this become `this._boundClearItems = clearItems.bind(this);`, but we will have to confirm with @stefanpenner or others ...